modules/network_synchronizer/data_buffer.cpp

```diff
@@ -135,14 +136,14 @@ void DataBuffer::dry() {
 }
 
 void DataBuffer::seek(int p_bits) {
-	ERR_FAIL_INDEX(p_bits, metadata_size + bit_size);
+	CRASH_BAD_INDEX(p_bits, metadata_size + bit_size);
```

Comment:
I would leave this as ERR_COND. This is not in direct control of the user, but if someone will use it, I think it's better to warn it, instead to crash the program, if he is doing some mistakes.